opm/input/eclipse/Schedule/Schedule.cpp

```diff
@@ -1453,8 +1464,15 @@ File {} line {}.)", pattern, location.keyword, location.filename, location.linen
         this->snapshots.resize(reportStep + 1);
         auto& input_block = this->m_sched_deck[reportStep];
         std::unordered_map<std::string, double> wpimult_global_factor;
+        ScheduleLogger logger(ScheduleLogger::select_stream(true, true),
+                              prefix, this->m_sched_deck.location());
+        
         for (auto& keyword : keywords) {
-            if (Action::PyAction::valid_keyword(keyword->name())) {
+            bool valid = Action::PyAction::valid_keyword(keyword->name());
+            if (this->lowActionParsingStrictness or valid) {
+                if (this->lowActionParsingStrictness and !valid) {
+                    logger(fmt::format("The keyword {} is not supported for inserting it from Python into a simulation, but you have set --action-parsing-strictness = low, so flow will try to apply the keyword still.", keyword->name()));
+                }
```

Comment:
This is  a bit unclear to me. I don't see the connection with ACTIONX. If this is also called from somewhere else we might get additional warning in te DEBUG file

Reply:
Or is this function only called when processing PYACTION?

Reply:
This function is only called from Python.